sound/soc/sof/ipc4-topology.c

```diff
@@ -81,10 +81,6 @@ static const struct sof_topology_token ipc4_out_audio_format_tokens[] = {
 		offsetof(struct sof_ipc4_audio_format, fmt_cfg)},
 };
 
-static const struct sof_topology_token ipc4_copier_gateway_cfg_tokens[] = {
-	{SOF_TKN_CAVS_AUDIO_FORMAT_DMA_BUFFER_SIZE, SND_SOC_TPLG_TUPLE_TYPE_WORD, get_token_u32, 0},
-};
-
```

Comment:
should we remove the SOF_TKN_CAVS_AUDIO_FORMAT_DMA_BUFFER_SIZE definition from the list of tokens, or is this still used somewhere?

Reply:
is this actually not used for deep-buffer @ranj063 ?